python/dolfinx/wrappers/mesh.cpp

```diff
@@ -242,6 +242,15 @@ void mesh(py::module& m)
                                    self.values().size(), self.values().data(), \
                                    py::none());                                \
                              })                                                \
+      .def(                                                                    \
+          "find_indices",                                                      \
+          [](dolfinx::mesh::MeshTags<SCALAR>& self, SCALAR i) {                \
+            std::vector<std::int32_t> indices_i = self.find_indices(i);        \
+            return py::array_t<std::int32_t>(indices_i.size(),                 \
+                                             indices_i.data());                \
+          },                                                                   \
+          "Return MeshTag indices having value i",                             \
+          py::return_value_policy::reference)                                  \
```

Comment:
I think you should just have automatic (default) rvp here. The automatic should be `move` here which is what you want.